controller/invitation_blackbox_test.go

```diff
@@ -245,6 +246,33 @@ func (s *TestInvitationREST) TestCreateOrganizationInvalidUserInvitation() {
 	require.Equal(s.T(), 0, len(invitations))
 }
 
+func (s *TestInvitationREST) TestAcceptInvitation() {
+	g := s.NewTestGraph()
+	team := g.CreateTeam()
+	invitee := g.CreateUser()
+	inv := g.CreateInvitation(team, invitee)
+
+	service, controller := s.SecuredController(s.testIdentity)
+
+	test.AcceptInviteInvitationTemporaryRedirect(s.T(), service.Context, service, controller, inv.Invitation().AcceptCode.String())
+
+	// The invitation should no longer be there after acceptance
+	_, err := s.Application.InvitationRepository().FindByAcceptCode(s.Ctx, s.testIdentity.ID, inv.Invitation().AcceptCode)
+	require.Error(s.T(), err)
+	require.IsType(s.T(), errors.NotFoundError{}, err)
+}
+
+func (s *TestInvitationREST) TestAcceptInvitationFailsForInvalidCode() {
```

Comment:
Would be nice to validate the response in case the same code is being re-used - could you please add a test for the same?

Reply:
Do you mean add a test to try and accept an invitation that has already been accepted?

Reply:
Yes

Reply:
Same as previous comment.